src/pages/Events/EventDoorCounter.tsx

```
@@ -0,0 +1,210 @@
+  const [events, setEvents] = useState<EventData[]>(INITIAL_EVENTS);
+  const [selectedEventId, setSelectedEventId] = useState<string>(INITIAL_EVENTS[0].id);
+  const [doorCount, setDoorCount] = useState<number>(0);
+  const [isRealtime, setIsRealtime] = useState<boolean>(true);
+
+  const selectedEvent = events.find(event => event.id === selectedEventId) || INITIAL_EVENTS[0];
+
+  // Simulate Real-Time Data Updates
+  useEffect(() => {
+    if (!isRealtime) return;
+    const interval = setInterval(() => {
+      setDoorCount(prev => Math.min(prev + 1, selectedEvent.capacity));
```

ESLint

[error] Replace `(prev` with `((prev)`

(prettier/prettier)